src/haddock/modules/topology/topocg/aa2cg.py

```diff
@@ -19,13 +19,15 @@
 import random
 import subprocess
 import warnings
+import logging
```

Comment:
Here you are using python's logger, it should be using haddock one instead

`from haddock import log`

And then adapt the calls in the code

Reply:
Done